js/common/lib/backend.ts

```diff
@@ -31,6 +31,26 @@ export interface SessionHandler {
       options: InferenceSession.RunOptions): Promise<SessionHandler.ReturnType>;
 }
 
+export declare namespace TrainingSessionHandler {
+  type FeedsType = {[name: string]: OnnxValue};
+  type FetchesType = {[name: string]: OnnxValue | null};
+  // shall we change this to just error code. Right now this will return loss, but it is not
+  // as important and too much cost to read the loss just for printing purpose.
+  type ReturnType = {[name: string]: OnnxValue}|number;
```

Comment:
shall we add this with runTrainStep api? Right now there is no usage of this so difficult to review.

Reply:
good idea -- removed from this PR for now